support-frontend/assets/components/marketingConsent/marketingConsentWithCheckbox.jsx

```diff
@@ -92,6 +93,24 @@ function MarketingButton(props: ButtonPropTypes) {
   );
 }
 
+const createRegionalTextVariations = (titleFreq: string, msgFreq: string): { titleFreq: string, msgFreq: string } => ({
+  titleFreq,
+  msgFreq,
+});
+
+const copy = {
+  ukRowRecurring: createRegionalTextVariations('a weekly email', 'from the week '),
```

Comment:
`createRegionalTextVariations` can be removed if you just return e.g. `{ titleFreq: 'a weekly email', msgFreq: 'from the week ' }`

Reply:
👍